[PATCH] configure.ac: Recognize linux-uclibc as well

Ltrace works on uClibc as well as on glibc, so accept it.

[Gustavo: update for ltrace 0.7.1]
Signed-off-by: Peter Korsgaard <jacmet@sunsite.dk>
Signed-off-by: Gustavo Zacarias <gustavo@zacarias.com.ar>

diff -Nura ltrace-0.7.1.orig/configure.ac ltrace-0.7.1/configure.ac
--- ltrace-0.7.1.orig/configure.ac	2012-12-03 09:02:32.995352741 -0300
+++ ltrace-0.7.1/configure.ac	2012-12-03 09:02:56.579096389 -0300
@@ -32,7 +32,7 @@
 AC_CANONICAL_HOST
 
 case "${host_os}" in
-    linux-gnu*)	HOST_OS="linux-gnu" ;;
+    linux-gnu* | linux-uclibc*)	HOST_OS="linux-gnu" ;;
     *)		AC_MSG_ERROR([unkown host-os ${host_os}]) ;;
 esac
 AC_SUBST(HOST_OS)
